### ISSUE TITLE
Slides are not loaded when swiping is fast.

### ISSUE THREAD

**Issue by noisik** [cubiq/SwipeView#42]
I change a little bit the css to view three slides at time (https://github.com/cubiq/SwipeView/issues/30). It works perfect, until...

The Problem: It happen in two ways (although its maybe the same thing):
1. When swiping faster to the right/left, some of the next slides aren't loaded or show (a blank space instead).
2. Without release the finger, swiping to right or left, an then released.

When a slide should load, well, it doesn't and the blank space is the only thing you see.

Not sure if its some problem with css or is because the way the .js work.

There is a way to load more slides at the page load and avoid loading one by one, or just a way of swiping without see the slides loading or not loading at all?

This is the demo with the problem: https://dl.dropbox.com/u/5700907/SCRO/swv_github/index.html

Snapshot here: https://dl.dropbox.com/u/5700907/SCRO/swv_github/img/snap_problem.png

**Comment by noisik** [comment]
Any Ideas? :/

**Comment by ghost** [comment]
This is actually a feature. "3) content is loaded only when needed, if you quickly swipe from page 1 to page 10, pages in between are ignored". In the gallery demo a blank image is shown when the swiping is fast so that the user understands content is not loaded. You should try to implement something similar in your version.

**Comment by noisik** [comment]
Yep, I understand that, but in this case, that's an usability problem and could let to confusion (people think there is an error loading the page or the images, and that's not good).

There's a workaround to make this?, it's posible to tweak o disable the feature?

Note: I did try show blank images, but that's the same, once the image is "ignore" it will remain that way, but instead, would be great that the image load when the slide slow the velocity, or at least when the move is finish.

**Comment by ghost** [comment]
I don't really agree with you on this being an usability problem. If you display a spinning wheel then users will get that at this speed content is not loaded. But anyway I'm not good enough to know where this comes from in the code, sorry.

**Comment by noisik** [comment]
But when the spinning wheel stop, they know it will show something, nothing will be missing or not loaded.

Thanks for answer, Im not good at js so I probably will ended doing something diferent.
